app/views/NewMessageView.tsx

```diff
@@ -153,7 +153,9 @@ class NewMessageView extends React.Component<INewMessageViewProps, INewMessageVi
 	};
 
 	search = async (text: string) => {
-		const result = await RocketChat.search({ text, filterRooms: false });
+		// TODO: When migrate rocketchat.js pass the param ISearch to there and the return should be
+		// ISearch | TSubscriptionModel, this because we do a local search too
+		const result = (await RocketChat.search({ text, filterRooms: false })) as unknown as ISearch[];
```

Comment:
A doubt, I saw in other files but I can't let it go.
why use `as unknown as ISearch`?
Is it some detail of the DB model?